Problem3.hs

```diff
@@ -0,0 +1,7 @@
+elementAt :: [a] -> Int -> a
+elementAt xs n = xs !! (n - 1)
+

```

Comment:
Try to come up with one more solution to this problem, as a hint you can create a function called,
`elementAt :: [a] -> Int -> a` that actually calls another function that has a slightly different type signature namely,
`actualElementAt :: ([a], Int) -> Int -> a`.

Reply:
On the other hand you could use an approach similar to the `myLength` function.